src/IssueViz.Security/OpenInIDE/Api/OpenInIDERequestHandler.cs

```diff
@@ -55,16 +80,71 @@ Task IOpenInIDERequestHandler.ShowHotspotAsync(IShowHotspotRequest request)
                 throw new ArgumentNullException(nameof(request));
             }
 
+            return ShowHotspotAsync(request);
+        }
+
+        private async Task ShowHotspotAsync(IShowHotspotRequest request)
+        {
+            logger.WriteLine(OpenInIDEResources.OpenHotspot_ProcessingRequest, request.ServerUrl, request.ProjectKey,
+                request.OrganizationKey ?? OpenInIDEResources.OpenHotspot_NullOrganization, request.HotspotKey);
+
+            // Always show the Hotspots tool window. If we can't successfully process the
+            // request we'll show a gold bar in the window
+            toolWindowService.Show(HotspotsToolWindow.ToolWindowId);
```

Comment:
`OpenInIDEFailureInfoBar` knows that it's attaching itself to the Hotspots ToolWindow, so I think it makes sense that it's also the one that opens it as well. Otherwise the logic is split: if we change the infobar to attach to a different window, we will need to remember to change this place as well.

Alternatively, we can change the infobar to receive the id of the toolwindow, so the orchestrator will be the only one who knows where the infobar is shown.

Reply:
I prefer the second option since we always show the hotspots window, even in success cases.

Reply:
I'll make that change in a separate PR.